syntax = "proto3";
package stream;





/*
ReplaceAlertStates返回
*/
message ReplaceAlertStatesResponse {
message States {
    /*
    状态数据id
    */
    string id = 1 ;
    /*
    状态数据版本
    */
    int32 version = 2 ;
}
        
    
    /*
    推送的状态数据列表
    */
    repeated States states = 1 ;
}

/*
ReplaceAlertStatesApi返回
*/
message ReplaceAlertStatesResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    stream.ReplaceAlertStatesResponse data = 4 ;
}

